docs/proposals/timeseries-partitioning-in-compactor.md

```diff
@@ -264,4 +295,128 @@ T2 partition 4 - Hash(timeseries label) % 4 == 3 && % 8 == 7
 T3 partition 8 - Hash(timeseries label) % 8 == 7
 ```
 
+### Compaction Partitioning Examples
```

Comment:
what about the case where all block are more than level 1, but there are some legacy blocks mixed with blocks compacted by partitioning compaction?